### ISSUE TITLE
Wrong HTTP code being returned for creation of a task

### ISSUE THREAD

**Issue by isabelcosta** [anitab-org/mentorship-backend#448]
**Describe the bug**

Task and Task comment creation returns 200 and not 201, so the success HTTP code appears to not be documented. The right code is 201, so it should be fixed in the code.

**To Reproduce**

Pre-requisite: Have an ongoing mentorship relation between 2 users. The mentorship relation has to be in the ACCEPTED state.

1. Go to 'localhost:5000' once you run the app, so you can see the Swagger UI documentation
2. Create a new task using `POST /mentorship_relation/{request_id}/task` service for a mentorship relation you already have (Similarly for Task comments).
3. See that the returned HTTP code is 200, and what is on the Responses description is 201

**Expected behavior**

In the Responses code for `POST /mentorship_relation/{request_id}/task` service, the successful HTTP code should be 201 instead of 200

Similarly response of Task Comment creation should return 201 instead of 200.

**Screenshots**

<img width="1068" alt="Screenshot 2020-02-15 at 19 06 31" src="https://user-images.githubusercontent.com/11148726/74616984-c1c6f180-5122-11ea-83b9-975ea7dc6aaf.png">

**Desktop (please complete the following information):**
 - OS: Mac OS
 - Browser Chrome
 - Version 🤷‍♀ 

**Additional context**

You just have to fix the error code which is documented

**Comment by sumit-badsara** [comment]
@isabelcosta Isn't 201 the appropriate status code for task comment creation as a new instance of comment is created. So, instead of changing the documentation, we can change the backend code for appropriate status code. Only if 201 is correct! :thinking:

**Comment by isabelcosta** [comment]
You are correct! @BadduCoder I just checked [here](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#2xx_Success) :)
Will soon change the issue description to reflect that

**Comment by srivastava9** [comment]
@isabelcosta  I have updated my pr can you have a look?

**Comment by isabelcosta** [comment]
@srivastava9 when you wish to contribute to an issue, please let us know in the issue before starting working on it :) I will check your PR once I get the chance to.

**Comment by srivastava9** [comment]
Ya sure,I will take that in mind from future thanks :)

**Comment by srivastava9** [comment]
@isabelcosta Do we have any gitter channel actually i am facing some issues while registering as a user.

**Comment by isabelcosta** [comment]
@srivastava9 we have [AnitaB.org Open Source Zulip](https://anitab-org.zulipchat.com/) where you can discuss the project and a specific stream [#mentorship-system](https://anitab-org.zulipchat.com/#narrow/stream/222534-mentorship-system) for it as well.
Please join Zulip so you can also see other discussions about the project :)

**Comment by sumit-badsara** [comment]
@isabelcosta Even Task creating returns 200, instead of 201. We can add this to current issue I guess.

**Comment by isabelcosta** [comment]
@BadduCoder i don't understand anymore, should it be 200 or 201? 😅 what should we add to the issue?

**Comment by sumit-badsara** [comment]
What I wanted to point out was `Creating Task` and `Creating Task Comment`, both should return 201 instead they both return 200.
But this issue only pointed out one out of these. It can be changed to include both the changes.

Creation of any resource should result in status code 201 (Be it task or task comment).
Does this make sense? I hope I am not over complicating this :sweat_smile:

**Comment by srivastava9** [comment]
@BadduCoder @isabelcosta Can I update the pr to also return 201 for creating task also?

**Comment by isabelcosta** [comment]
> What I wanted to point out was `Creating Task` and `Creating Task Comment`, both should return 201 instead they both return 200.
> But this issue only pointed out one out of these. It can be changed to include both the changes.
> 
> Creation of any resource should result in status code 201 (Be it task or task comment).
> Does this make sense? I hope I am not over complicating this 😅

@BadduCoder Makes total sense :) thank you for clarifying, I learned something new here :)
I think you have write access to change the issue description, or am I wrong? I can change though

**Comment by isabelcosta** [comment]
> @BadduCoder @isabelcosta Can I update the pr to also return 201 for creating task also?

Sure, do that as @BadduCoder said ;)

**Comment by sumit-badsara** [comment]
@isabelcosta Updated the PR comment. :+1:

**Comment by srivastava9** [comment]
I have made the pr could you please review it? Changes as suggested by @BadduCoder such as updating docs and test was also there

**Comment by srivastava9** [comment]
I have made the pr could you please review it.Changes suggested by @BadduCoder such as updating docs and test cases have already been made.
@isabelcosta @BadduCoder Could you please review it?
